rubix-common/src/main/java/com/qubole/rubix/common/utils/ClusterUtil.java

```diff
@@ -85,4 +89,14 @@ public static Configuration applyRubixSiteConfig(Configuration conf)
 
     return conf;
   }
+
+  public static Set<BookkeeperMetricsReporter> getMetricReportors(Configuration configuration)
```

Comment:
we have **Metrics** BookkeeperMetricsReporter but in class name we have used **Metric**. Please change it to Metrics to remove any ambiguity.